bookkeeper-dist/src/main/resources/LICENSE-bkctl.bin.txt

```diff
@@ -0,0 +1,418 @@
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+------------------------------------------------------------------------------------
+The following bundled 3rd party jars are distributed under the
+Apache Software License, Version 2.
+
+- lib/com.fasterxml.jackson.core-jackson-annotations-2.9.7.jar [1]
+- lib/com.fasterxml.jackson.core-jackson-core-2.9.7.jar [2]
+- lib/com.fasterxml.jackson.core-jackson-databind-2.9.7.jar [3]
+- lib/com.google.guava-guava-21.0.jar [4]
+- lib/commons-cli-commons-cli-1.2.jar [5]
+- lib/commons-codec-commons-codec-1.6.jar [6]
+- lib/commons-configuration-commons-configuration-1.10.jar [7]
+- lib/commons-io-commons-io-2.4.jar [8]
+- lib/commons-lang-commons-lang-2.6.jar [9]
+- lib/commons-logging-commons-logging-1.1.1.jar [10]
+- lib/io.netty-netty-buffer-4.1.31.Final.jar [11]
+- lib/io.netty-netty-codec-4.1.31.Final.jar [11]
+- lib/io.netty-netty-codec-http-4.1.31.Final.jar [11]
+- lib/io.netty-netty-codec-http2-4.1.31.Final.jar [11]
+- lib/io.netty-netty-codec-socks-4.1.31.Final.jar [11]
+- lib/io.netty-netty-common-4.1.31.Final.jar [11]
+- lib/io.netty-netty-handler-4.1.31.Final.jar [11]
+- lib/io.netty-netty-handler-proxy-4.1.31.Final.jar [11]
+- lib/io.netty-netty-resolver-4.1.31.Final.jar [11]
+- lib/io.netty-netty-tcnative-boringssl-static-2.0.19.Final.jar [11]
+- lib/io.netty-netty-transport-4.1.31.Final.jar [11]
+- lib/io.netty-netty-transport-native-epoll-4.1.31.Final.jar [11]
+- lib/io.netty-netty-transport-native-unix-common-4.1.31.Final.jar [11]
+- lib/log4j-log4j-1.2.17.jar [16]
+- lib/net.java.dev.jna-jna-3.2.7.jar [17]
+- lib/org.apache.commons-commons-collections4-4.1.jar [18]
+- lib/org.apache.commons-commons-lang3-3.6.jar [19]
+- lib/org.apache.zookeeper-zookeeper-3.4.13.jar [20]
+- lib/org.rocksdb-rocksdbjni-5.13.1.jar [22]
```

Comment:
We could exclude this as it's only relevant to bookies

Reply:
good point; excluded rocksdb